pom.xml

```diff
@@ -14,7 +14,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>gov.nasa.podaac</groupId>
   <artifactId>cumulus-metadata-aggregator</artifactId>
-  <version>8.7.0-alpha.7-SNAPSHOT</version>
+  <version>8.8.0-alpha.0-SNAPSHOT</version>
```

Comment:
If merging into master this should be 8.8.0 also right and not a snapshot/alpha right?
master branch being "final" / "official" and all